### DISCUSSION TITLE
Valheim error when starting the server

### DISCUSSION THREAD

**Discussion by Pawau144** [GameServerManagers/LinuxGSM#3255] [Q&A] [answered]
I do get an error when I try to start the server:
[ FAIL ] Starting vhserver: executable was not found
* /home/vhserver/serverfiles/valheim_server.x86_64

I installed all from your tutorial https://linuxgsm.com/lgsm/vhserver/

Can you guys help me please? :)

**Comment by EugeneGGHQ** [comment]
Hi Pawau,

Couple of things to check:

1. Can you confirm that the file /home/vhserver/serverfiles/valheim_server.x86_64 definitely exists
          (If it isn't, please try running ./vhserver validate ) 
2. Confirm that you are launching as the vhserver user
3. Check the current permissions on the file using `ls -la` in the /home/vhserver/serverfiles directory.

**Comment by Pawau144** [comment, marked as answer]
I got it running now. Had to reset the root server because steamcmd wasn’t installed correctly.
